tests/multi_ptr/multi_ptr_convert_assignment_ops.h

```diff
@@ -0,0 +1,124 @@
+                  const std::string &src_is_decorated_name,
+                  const std::string &dst_address_space_name,
+                  const std::string &dst_is_decorated_name) {
+    auto queue = sycl_cts::util::get_cts_object::queue();
+    T value = user_def_types::get_init_value_helper<T>(expected_val);
+    SECTION(
+        section_name(
+            "Check &operator=(const multi_ptr<value_type, ASP, IsDecorated>&)")
+            .with("T", type_name)
+            .with("src address_space", src_address_space_name)
+            .with("src decorated", src_is_decorated_name)
+            .with("dst address_space", dst_address_space_name)
+            .with("dst decorated", dst_is_decorated_name)
+            .create()) {
+      bool res = false;
+      {
+        sycl::buffer<bool, 1> res_buf(&res, sycl::range<1>(1));
+        sycl::buffer<T> val_buffer(&value, sycl::range<1>(1));
+        queue.submit([&](sycl::handler &cgh) {
+          auto res_acc =
+              res_buf.template get_access<sycl::access_mode::write>(cgh);
+          auto acc_for_mptr =
+              val_buffer.template get_access<sycl::access_mode::read>(cgh);
+          cgh.single_task([=] {
+            const src_multi_ptr_t mptr_in(acc_for_mptr);
+            dst_multi_ptr_t mptr_out;
+
+            mptr_out = mptr_in;
+
+            // Check that second mptr has the same value as first mptr
+            res_acc[0] = *(mptr_out.get_raw()) == acc_for_mptr[0];
+          });
+        });
+      }
+      CHECK(res);
+    }
+
+    SECTION(
+        section_name(
+            "Check &operator=(multi_ptr<value_type, ASP, IsDecorated>&)")
+            .with("T", type_name)
+            .with("src address_space", src_address_space_name)
+            .with("src decorated", src_is_decorated_name)
+            .with("dst address_space", dst_address_space_name)
+            .with("dst decorated", dst_is_decorated_name)
+            .create()) {
+      bool res = false;
+      {
+        sycl::buffer<bool, 1> res_buf(&res, sycl::range<1>(1));
+        sycl::buffer<T> val_buffer(&value, sycl::range<1>(1));
+        queue.submit([&](sycl::handler &cgh) {
+          auto res_acc =
+              res_buf.template get_access<sycl::access_mode::write>(cgh);
+          auto acc_for_mptr =
+              val_buffer.template get_access<sycl::access_mode::read>(cgh);
+          cgh.single_task([=] {
+            src_multi_ptr_t mptr_in(acc_for_mptr);
+            dst_multi_ptr_t mptr_out;
+
+            mptr_out = mptr_in;
+
+            // Check that second mptr has the same value as first mptr
+            res_acc[0] = *(mptr_out.get_raw()) == acc_for_mptr[0];
+          });
+        });
+      }
+      CHECK(res);
+    }
+    }
+};
+
+template <typename T>
+class check_multi_ptr_convert_assign_for_type {
+ public:
+  void operator()(const std::string &type_name) {
+    const auto address_spaces = multi_ptr_common::get_address_spaces();
```

Comment:
Where are these functions in this PR?

Reply:
These functions can be found [here, in PR#346](https://github.com/KhronosGroup/SYCL-CTS/pull/346/files#diff-25d748f355322c69a0d0c71d0ac575add28d8f32938e0e7642adf4c9679f3936R17-R37)